LoopKit/GlucoseKit/GlucoseStore.swift

```diff
@@ -434,6 +462,25 @@ extension GlucoseStore {
             }
         }
     }
+    
+    private func deleteCachedObjects(forSampleUUIDs uuids: [UUID], batchSize: Int = 500) -> Int {
+        dispatchPrecondition(condition: .onQueue(dataAccessQueue))
+
+        var deleted = 0
+
+        for batch in uuids.chunked(into: batchSize) {
+            let result = self.purgeCachedGlucoseObjects(matching: NSPredicate(format: "uuid IN %@", batch.map { $0 as NSUUID }))
+            switch result {
+            case .failure:
+                // ignore for now; it's already logged
+                break
+            case .success(let count):
+                deleted += count;
```

Comment:
[nit] Trailing semi-colon not necessary.

Reply:
Must've been on one of those days I was writing javascript too. :)